payment_paybox/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Paybox payment acquirer module for Odoo
+=======================================
+
+This module adds a new payment acquirer for your website sales: Paybox,
+provided by Verifone.  Only useful for French organizations.
+
+Installation
+============
+
+Requires the `Python Crypto module <http://www.pycrypto.org/>`_. Make sure to
+run ``apt-get install python-crypto`` before installing this module.
+
+Usage
+=====
+
+This modules installs with default values so that you can reach the Paybox
+payment screen.  For proper tests, also install the ``website_sale`` module.  Go
+to the Payment Acquirers configuration screen to set the return URL of your
+Odoo installation to properly process the payment.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
```

Comment:
You should replace `{project_repo}` and the other place_holders with the proper values